### PR TITLE
Remove the all target and update the help.

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jaycee** [comment]
There is no QA beyond reviewing the code.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2822/
Test PASSed.

File: Makefile [hunk 1]
```diff
@@ -44,13 +44,18 @@ STATIC_CSS_FILES = \
...
 	@echo "check - run tests and check lint."
 	@echo "clean - remove build and python artifacts"
+	@echo "clean-gui - clean the built gui js code"
+	@echo "clean-downloadcache - remove the downloadcache"
 	@echo "deps - install the dependencies"
-	@echo "dev - install jujugui in develop mode"
 	@echo "gui - build the gui files"
-	@echo "lint - check style with flake8"
+	@echo "lint - check python style with flake8"
+	@echo "lint-js - check javascript style with elint"
```
Comment (huwshimi) [inline Makefile]: With "e**s**lint"?

**Comment by huwshimi** [comment]
:+1: with one question. Thanks @jaycee!

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2870/
Test PASSed.

**Comment by jaycee** [comment]
Thanks @huwshimi. This is trivial, so :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**